src/update-handling/FileService.ts

```diff
@@ -129,8 +129,11 @@ export class FileService {
             }
 
             fileContent.push("[" + h.hotkeyCommand + "]");
-            const hotKey = h.isStagedUpgrade ? `${h.hotKey},${h.hotKey},${h.hotKey}` : h.hotKey;
-            fileContent.push("Hotkey=" + hotKey);
+            if (h.hotKey) {
```

Comment:
auras don't have "Hotkey" so we were printing "Hotkey=undefined". added a check to prevent this.